other/cmr-exchange/service-bridge/resources/docs/rest-api/index.md

```diff
@@ -179,12 +179,19 @@ and, the time taken to process the request.
 {
   "hits": ...,
   "took": ...,
+  "sa-header": "[...]",
   "items": [...],
   "request-id": ...,
   "warnings:" null
 }
 ```
 
+## Search-After
+
+CMR Service-Bridge allows the use of CMR-Search-After. When searching a collection
```

Comment:
Maybe change to something like:

"CMR Service-Bridge allows the use of CMR-Search-After header to take advantage of the CMR Search-After support for paging. Any search against CMR Service Bridge that has results not fully returned in the current request will return a search-after value in the CMR-Search-After header of the search response. User can then pass this returned value in the CMR-Search-After header of the following request to retrieve the next page of result based on the specified page_size. See CMR API document on Search-After for a detailed description."

In your description, it says `the response will include the Search-After header as a header
and in the response body.`. Will the search-after header be returned in the response body also? Why do we want to do that?

Reply:
Unfortunately, with how some of the service-bridge/cmr-exchange stuff is set up, the data where the response headers are set passed through an area of code that adds values to the response body. I tried passing the data through without adding it to the response, but I don't think the code is currently set up in a good way for handling that. the headers CMR-Hits, CMR-Took, and CMR-Request-ID all go through the same thing (ending up in header and response body)

Reply:
With Liu's help, it is now removed from the response body